app/common/common-module.js

```diff
@@ -41,6 +41,7 @@ angular.module('ushahidi.common', [
 .controller('navigation', require('./controllers/navigation.js'))
 .controller('PageMetadata', require('./controllers/page-metadata.js'))
 .controller('notifier', require('./controllers/notifier.js'))
+.controller('intercom', require('./controllers/intercom.js'))
```

Comment:
Yeah that does make sense. BUT at the moment it's based on login, if the user can't login - if the app fails completely - then we have no way to identify them.

Would it make sense to have an outer intercom that kicks in on failed load that sits outside of angular?